development/webpack/test/cli.test.ts

```diff
@@ -0,0 +1,82 @@
+import { describe, it } from 'node:test';
```

Comment:
I used `node:test`. Thoughts?

Reply:
Only because I generally prefer to use the "standard library" when its suitable.